shell/common/persistent_cache.cc

```diff
@@ -81,6 +81,7 @@ bool PersistentCache::IsValid() const {
 // |GrContextOptions::PersistentCache|
 sk_sp<SkData> PersistentCache::load(const SkData& key) {
   TRACE_EVENT0("flutter", "PersistentCacheLoad");
+  is_accessed_ = true;
```

Comment:
Aren't we only interested in pictures where a persistent cache entry is stored? In cases of cache loads, we are already reusing something that we have seen (and cached) before. My anticipation of the logic was:

1. Skia asks us to store shaders for a new picture we have never seen before.
2. We store the skp for that picture.

Reply:
This about this some more. Maybe you didn't mean for this feature to say "give me the SKP dumps of new pictures that cause shader compilations", but instead, "give me all SKP dumps of pictures that already caused shader compilations". If the latter is the case, just look for all SKPs on disk. Either way, the `is_accessed` logic I think can be made more elegant.

Reply:
Done. Changed the flag to switch on store instead on load.